addons/hr/views/hr_employee_views.xml

```diff
@@ -231,7 +237,7 @@
             <field name="name">hr.employee.tree</field>
             <field name="model">hr.employee</field>
             <field name="arch" type="xml">
-                <tree string="Employees" multi_edit="1" sample="1" js_class="hr_employee_list">
+                <tree string="Employees" expand="context.get('expand', False)" multi_edit="1" sample="1" js_class="hr_employee_list">
```

Comment:
I'd use a less generic name than `expand` as it can be confusing and not know where/why it's used.
Something like `expand_employee_list` maybe